web/core/components/workspace-notifications/sidebar/filters/menu/root.tsx

```
@@ -3,6 +3,7 @@
 import { FC } from "react";
 import { observer } from "mobx-react";
 import { ListFilter } from "lucide-react";
+import { useTranslation } from "@plane/i18n";
 import { PopoverMenu, Tooltip } from "@plane/ui";
 import { useTranslation } from "@plane/i18n";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)